components/enrichment-chart/index.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import {
+  BarChart,
+  Bar,
+  LabelList,
+  XAxis,
+  YAxis,
+  ResponsiveContainer,
+} from 'recharts'
+
+const VerticalBarChart = ({
+  data,
+  dataKey = 'value',
+  className,
+  width = '100%',
+  height = 128,
+  ...restProps
+}) => (
+  <ResponsiveContainer className={className} width={width} height={height}>
+    <BarChart data={data} layout="vertical" {...restProps}>
+      <XAxis type="number" hide />
+      <YAxis type="category" hide />
+      <Bar dataKey={dataKey} isAnimationActive={false}>
+        <LabelList
+          dataKey="caption"
+          position="insideLeft"
+          fill="#fff"
+          offset={16}
+        />
+        <LabelList
+          dataKey="value"
+          position="insideRight"
+          formatter={value => Number(value).toLocaleString('en-GB')}
+          fill="#fff"
+          offset={16}
+        />
+      </Bar>
+    </BarChart>
+  </ResponsiveContainer>
+)
+
+const EnrichmentChart = ({ value, caption, increase }) => {
```

Comment:
Can you please put here check for a null value. Otherwise, if you go directly to https://dashboard-git-dois.oacore.now.sh/data-providers/27/doi it will fail because of some sort of recursion problem in recharts library. (They expect that props are correct)

```
const EnrichmentChart = ({ value, caption, increase }) => {
  if (!value) return null
  const data = [
    increase && {
      caption: 'CORE',
      value: value + increase,
      fill: 'var(--primary)',
    },
    {
      caption,
      value,
      fill: increase ? 'var(--gray-700)' : 'var(--primary)',
    },
  ].filter(truthy => truthy)

  return <VerticalBarChart data={data} />

```